src/scripts/createMarkdown.sh

```diff
@@ -81,43 +81,148 @@ Feature: Responsive Design
     Then content remains readable without loss of information
 ```
 
-# Criteria
+## Criteria
```

Comment:
maybe we should remove? I am thinking we only need condensed and gherkin for all entries